include/mrdox/MetadataFwd.hpp

```diff
@@ -42,8 +42,11 @@ struct Reference;
 struct Scope;
 struct SymbolInfo;
 struct TemplateInfo;
+struct TypeTParamInfo;
+struct NonTypeTParamInfo;
+struct TemplateTParamInfo;
 struct TemplateParamInfo;
-struct TemplateSpecializationInfo;
+struct TemplateArgInfo;
```

Comment:
Please group all the template related fwd declarations together in their own section below, separated by a blank line above and below